pkg/apiserver/info/info.go

```diff
@@ -110,6 +94,37 @@ func (s *Service) infoHandler(c *gin.Context) {
 	c.JSON(http.StatusOK, resp)
 }
 
+func (s *Service) checkNgmState() (utils.NgmState, error) {
+	ngmState := utils.NgmStateNotSupported
+
+	// The FeatureVersion value is "N/A" when running `make && bin/tidb-dashboard`
+	// The FeatureVersion value is "Unknown" when running `go run cmd/tidb-dashboard/main.go`
+	featureSupported :=
```

Comment:
It's interesting, why do we have these two different values? Is it possible to be simply `""`, which is the well-known default value for a missing string?

Reply:
The config.FeatureVersion is assigned to version.PDVersion default.

```
func Default() *Config {
	return &Config{
		DataDir:            "/tmp/dashboard-data",
		TempDir:            "",
		PDEndPoint:         "http://127.0.0.1:2379",
		PublicPathPrefix:   defaultPublicPathPrefix,
		ClusterTLSConfig:   nil,
		TiDBTLSConfig:      nil,
		EnableTelemetry:    true,
		EnableExperimental: false,
		FeatureVersion:     version.PDVersion,
	}
}
```

and the version.PDVeresion default value is "Unknown".

```
// Version information. It will be overwritten by LDFLAGS.
var (
	InternalVersion = "Unknown"
	Standalone      = "Unknown" // Unknown, Yes or No
	PDVersion       = "Unknown"
	BuildTime       = "Unknown"
	BuildGitHash    = "Unknown"
)
```

Its value is overwritten to "N/A" by LDFLAGS when building in dev mode.

```
LDFLAGS += -X "$(DASHBOARD_PKG)/pkg/utils/version.PDVersion=N/A"
```

Let me try to assign to `""` if PDVersion is "N/A" and "Unknown".

Reply:
Can we unify the PDVersion and the FEATURE_VERSION? Maybe we can specify the pd version during development.

Reply:
How about not assigning `version.PDVersion` to `Default()`? Will it cause any problems?

Reply:
> How about not assigning `version.PDVersion` to `Default()`? Will it cause any problems?

Nope, my initial implementation is just to assign the FeatureVersion `""` default value, but later we need to use `FeatureVersion` and `version.PDVersion` both, to decide which version is preferable to use.

Reply:
> How about not assigning `version.PDVersion` to `Default()`? Will it cause any problems?

I think it should be the same effect. we eventually need to use the `version.PDVersion`, before or later.